core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -487,8 +487,8 @@ private[spark] class TaskSetManager(
           // a good proxy to task serialization time.
           // val timeTaken = clock.getTime() - startTime
           val taskName = s"task ${info.id} in stage ${taskSet.id}"
-          logInfo("Starting %s (TID %d, %s, %s, %d bytes)".format(
-              taskName, taskId, host, taskLocality, serializedTask.limit))
+          logInfo(s"Starting $taskName (TID $taskId, $host, ${task.partitionId}," +
+            s"$taskLocality, ${serializedTask.limit} bytes)")
```

Comment:
I like the inclusion of the partitionId in the msg, but can you add a "partition" label in there, eg

```scala
logInfo(s"Starting $taskName (TID $taskId, $host, partition ${task.partitionId}," +
            s"$taskLocality, ${serializedTask.limit} bytes)")
```